index.d.ts

```diff
@@ -4,13 +4,14 @@ declare module 'evergreen-ui' {
   import { IconName } from '@blueprintjs/icons'
   import * as React from 'react'
   import { extractStyles as boxExtractStyles } from 'ui-box'
-  import { BoxProps } from 'ui-box/dist/types/box-types'
+  import { BoxProps, Is } from 'ui-box/dist/types/box-types'
```

Comment:
Maybe `ui-box` should properly export these types so you can just do:
```
import { BoxProps, Is } from 'ui-box'
```

Reply:
Yes! Would be better.